Scripts/Andrew01.swift

```diff
@@ -11,14 +11,11 @@ import PenguinStructures
 
 /// Andrew01: RAE Tracker
 struct Andrew01: ParsableCommand {
-  @Option(help: "Run on track number x")
-  var trackId: Int = 0
-  
   @Option(help: "Run for number of frames")
   var trackLength: Int = 80
   
   @Option(help: "Size of feature space")
-  var featureSize: Int = 5
+  var featureSize: Int = 256
```

Comment:
This would make a 256x256 covariance matrix, what is the time cost compared to, for example, 16-d feature?

Reply:
It is quite slow, but significantly increases model performance. I can run a quick time test if you'd like.

Reply:
~8.5 seconds to infer each frame for 256-d feature vs. ~8.2 seconds to infer each from for 16-d feature. I guess this part is not the bottleneck?

Reply:
This is very interesting. Maybe later we can do some profiling and see why is this the case.